aptos-move/block-executor/src/view.rs

```diff
@@ -364,6 +365,33 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
         })
     }
 
+    /// Given a state value, performs deserialization-serialization round-trip
+    /// to replace any aggregator / snapshot values.
+    pub(crate) fn replace_identifiers_with_values(
+        &self,
+        bytes: &Bytes,
+        layout: &MoveTypeLayout,
+    ) -> anyhow::Result<(Bytes, Vec<T::Identifier>)> {
+        // TODO: Find a way to replace this with Self::IdentifierV2
```

Comment:
Aren't these two the same?

Reply:
They are the same. I just thought you don't like repeated code ;)